docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,222 @@ license: |
+does not support moving the views across databases.
+
+#### Syntax
+{% highlight sql %}
+ALTER VIEW view_identifier RENAME TO view_identifier
+{% endhighlight %}
+
+#### Parameters
+<dl>
+  <dt><code><em>view_identifier</em></code></dt>
+  <dd>
+    Specifies a view name, which may be optionally qualified with a database name.<br><br>
+    <b> Syntax:</b>
+      <code>
+        [database_name.]view_name
+      </code>
+  </dd>
+</dl>
+
+#### SET View Properties
+Set one or more properties of an existing view. The properties are the key value pairs. If the properties' keys exist, 
+the values are replaced with the new values. If the properties' keys do not exist, the key value pairs are added into
+the properties.
+
+#### Syntax
+{% highlight sql %}
+ALTER VIEW view_identifier SET TBLPROPERTIES
+  (property_key=property_val [, ...])
+{% endhighlight %}
+
+#### Parameters
+<dl>
+  <dt><code><em>view_identifier</em></code></dt>
+  <dd>
+    Specifies a view name, which may be optionally qualified with a database name.<br><br>
+    <b> Syntax:</b>
+      <code>
+        [database_name.]view_name
+      </code>
+  </dd>
+  <dt><code><em>property_key</em></code></dt>
+  <dd>
+    Specifies the property key. The key may consists of multiple parts separated by dot.<br><br>
+    <b>Syntax:</b>
+      <code>
+        [key_part1][.key_part2][...]
+      </code>
+  </dd>
+</dl>
+
+#### UNSET View Properties
+Drop one or more properties of an existing view. If the specified keys do not exist, an exception is thrown. Use 
+`IF EXISTS` to avoid the exception. 
+
+#### Syntax
+{% highlight sql %}
+ALTER VIEW view_identifier UNSET TBLPROPERTIES [IF EXISTS]
+  (property_key [, ...])
+{% endhighlight %}
+
+#### Parameters
+<dl>
+  <dt><code><em>view_identifier</em></code></dt>
+  <dd>
+    Specifies a view name, which may be optionally qualified with a database name.<br><br>
+    <b> Syntax:</b>
+      <code>
+        [database_name.]view_name
+      </code>
+  </dd>
+  <dt><code><em>property_key</em></code></dt>
+  <dd>
+    Specifies the property key. The key may consists of multiple parts separated by dot.<br><br>
+    <b>Syntax:</b>
+      <code>
+        [key_part1][.key_part2][...]
+      </code>
+  </dd>
+</dl>
+
+#### ALTER View AS SELECT
+`ALTER VIEW view_identifier AS SELECT` statement changes the definition of a view, the `SELECT` statement must be valid,
+and the `view_identifier` must exist.
+
+#### Syntax
+{% highlight sql %}
+ALTER VIEW view_identifier AS select_statement
+{% endhighlight %}
+
+#### Parameters
+<dl>
+  <dt><code><em>view_identifier</em></code></dt>
+  <dd>
+    Specifies a view name, which may be optionally qualified with a database name.<br><br>
+    <b> Syntax:</b>
+      <code>
+        [database_name.]view_name
+      </code>
+  </dd>
+  <dt><code><em>select_statement</em></code></dt>
+  <dd>
+    Specifies the definition of the view, detail check [select_statement](sql-ref-syntax-qry-select.html)
```

Comment:
markdown link doesn't work inside html. I think you have to use the html link. Something like
<a href="sql-ref-syntax-qry-select.html">select_statement</a>

Reply:
@huaxingao thanks, let me change.